evalml/automl/automl_search.py

```diff
@@ -383,6 +383,8 @@ def __init__(self,
 
         if self.allowed_pipelines == []:
             raise ValueError("No allowed pipelines to search")
+        else:
+            logger.info("Pipelines generated.")
```

Comment:
Is there anything else we can tell people here? Perhaps `f"{len(self.allowed_pipelines)} pipelines ready for search."`